src/Tracker/onfidoTracker.tsx

```diff
@@ -87,6 +89,7 @@ export const sendAnalyticsEvent = (
   }
   const identificationProperties = {
     applicant_uuid,
+    anonymous_id: device_uuid,
```

Comment:
Given this is a uuid, maybe we should rename this `anonymous_uuid`. Might be worth bringing it up with See Wah, Petter and Javad